vendor.conf

```diff
@@ -23,6 +23,13 @@ github.com/docker/libnetwork 3ab699ea36573d98f481d233c30c742ade737565
 github.com/opencontainers/runc 8e8d01d38d7b4fb0a35bf89b72bc3e18c98882d7
 github.com/opencontainers/go-digest a6d0ee40d4207ea02364bd3b9e8e77b9159ba1eb
 
+github.com/docker/containerd 1dc5d652ac1adf8f0a92ee8eff7af07b129d4e21
+github.com/docker/libtrust 9cbd2a1374f46905c68a4eb3694a130610adc62a
+github.com/gorilla/context v1.1
```

Comment:
`github.com/docker/containerd` was needed for the gRPC stuff. I think @aaronlehmann is correct that most of the rest came in due to the need for manifest stuff from `distribution`. @stevvooe is that something which the content store functionality you are adding will address or shall I look for alternatives?

As an aside, figuring out the container config for a v1 schema (not the manifest itself, but the bit with `Cwd`, `Entrypoint`, `Cmd` etc in it) was basically guess work and reverse engineering, if anyone has an actual reference (to docs, or even better usable library code) I'd be glad to have them...